src/store/domains/globalSearch.ts

```diff
@@ -121,12 +121,14 @@ export default class GlobalSearch implements GlobalSearchStoreInterface, Routing
 
   setDatingFrom(fromYear: number) {
     this.filters.dating.fromYear = fromYear;
+    this.resetPagination();
```

Comment:
Vllt. macht es Sinn `resetPagination()` nur aus der `triggerFilterRequest()` (vor oder nach der `setSearchResult()`) oder innerhalb der `setSearchResult()` aufzurufen.
Sonst muss bei jedem weiteren Filter der dazu kommt, die `resetPagination` mit angegeben werden.
Dabei sollte es ja so sein, dass erst wenn vom Backend neue Results kommen, `from` zurückgesetzt wird.

Reply:
hatte ich auch schon … dann landete der in einer unendlich-schleife … vielleicht hast Du ne Idee, wie man selbiger Herr werden könnte :)

Reply:
Hast recht.

Dann muss `from` in bestimmten Actions vorher zurückgesetzt werden.
Habe in dem Zuge ein wenig refactored.

In der `search-result-navigation` gab es Referenzen auf GlobalSearch-Store-Felder, von denen ich sagen würde, dass sie recht Store- und Suchspezifisch sind.
Habe deshalb einige Variablen/Konstanten sowie Funktionen aus der Komponente in den Store verschoben und lasse sie von der Komponente aus abfragen bzw. aufrufen.

Das Ziel war es auch, die verschiedenen Berechnungen um `from`, `size` und `hits` herum aus der Komponente und am besten in den Store zu ziehen.
In der Komponente arbeitet man dann rein mit dem Konzept "Ergebnisseiten".

Hoffe das ist so ok.